test/date_range_test.rb

```diff
@@ -35,32 +35,32 @@
       expect(@range.overlap?(test_range)).must_equal true
```

Comment:
This is a good test for a single test case. The test cases below also need to be completed to insure that `.overlap?` returns the correct value for the different situations described below.